sdks/python/apache_beam/io/iobase.py

```diff
@@ -258,12 +258,23 @@ class DynamicSplitResult(object):
 
 
 class DynamicSplitResultWithPosition(DynamicSplitResult):
+  """"""
 
   def __init__(self, stop_position):
     assert isinstance(stop_position, ReaderPosition)
     self.stop_position = stop_position
 
 
+class BoundedSourceSplit(DynamicSplitResult):
+  """"""
```

Comment:
What sources would return this? I thought all dynamic splits were going to be in terms of positions, which is future-proof with SDF.